util/compression.h

```diff
@@ -14,7 +14,10 @@
 #ifdef ROCKSDB_MALLOC_USABLE_SIZE
 #ifdef OS_FREEBSD
 #include <malloc_np.h>
-#else  // OS_FREEBSD
```

Comment:
If it's not necessary for the original code to do this, can we just `#include "port/malloc.h"` instead?